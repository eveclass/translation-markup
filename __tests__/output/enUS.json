{
  "CREDIT_CARD": {
    "NAME": "Credit Card",
    "FLAG": {
      "VISA": "Visa",
      "AMEX": "American Express",
      "DINERS": "Diners",
      "MASTERCARD": "Mastercard"
    }
  }
}