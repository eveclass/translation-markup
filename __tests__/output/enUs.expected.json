{
  "COUNTRY": {
    "USA": {
      "STATES": {
        "NEW_YORK": {
          "NAME": "New York",
          "ACRONYM": "NY"
        },
        "CALIFORNIA": {
          "NAME": "California",
          "ACRONYM": "CA"
        },
        "NORTH_CAROLINA": {
          "NAME": "North Carolina",
          "ACRONYM": "NC"
        }
      },
      "NAME": "United States",
      "LANGUAGE": "English"
    },
    "SPAIN": {
      "NAME": "Spain",
      "LANGUAGE": "Spanish"
    },
    "BRAZIL": {
      "NAME": "Brazil",
      "LANGUAGE": "Portuguese"
    }
  },
  "MENU": {
    "CART": {
      "EMPTY": "Empty Cart",
      "CHECKOUT": "Checkout"
    },
    "USER": {
      "LABEL": "User",
      "DROPDOWN": {
        "EDIT": "Edit",
        "LOGOUT": "Logout"
      }
    }
  }
}